ecosystem/sep-0012.md

```diff
@@ -37,7 +37,8 @@ the API endpoints according to the protocol will work with any compliant wallet.
 - An anchor must define the location of their `KYC_SERVER` or `TRANSFER_SERVER` in their [`stellar.toml`](sep-0001.md).
   This is how a client app knows where to find the anchor's server. A client app will send KYC requests to the
   `KYC_SERVER` if it is specified, otherwise to the `TRANSFER_SERVER`.
-- Anchors and clients must support [SEP-10](sep-0010.md) web authentication and use it for all SEP-12 endpoints.
+- Anchors and clients must support [SEP-10](sep-0010.md) and optionally [SEP-45](sep-0045.md) web authentication and use
```

Comment:
According to [SEP-45](https://github.com/stellar/stellar-protocol/pull/1623), KYC may be updated by SEP-10 or SEP-45. 

```suggestion
- Anchors and clients must support one of [SEP-10](sep-0010.md) and [SEP-45](sep-0045.md) web authentication and use
```